sdk/ml/azure-ai-ml/azure/ai/ml/entities/_monitoring/thresholds.py

```diff
@@ -95,13 +194,47 @@ def __init__(
             MonitorMetricName.MAE,
             MonitorMetricName.MSE,
             MonitorMetricName.RMSE,
-        ] = None,
+        ],
         threshold: float = None,
     ):
         super().__init__(threshold=threshold)
-        self.applicable_feature_type = MonitorFeatureType.NOT_APPLICABLE
         self.metric_name = metric_name
 
+    def _to_rest_object(self, **kwargs) -> ModelPerformanceMetricThresholdBase:
```

Comment:
This might benefit from Python's match/case construct, once the min version of the SDK reaches 3.10.

Reply:
for sure, I'm also hoping to get the service folks/PM to agree on the same enum values for SDK/CLI/REST to make life easier in the next API version